wiki/osu!wiki_Contribution_Guide/id.md

```diff
@@ -0,0 +1,106 @@
+
+Jika, selama bagian apa pun, anda bingung dan/atau memerlukan bantuan lebih lanjut, jangan ragu mengirim pesan ke [Discord osu!dev](https://discord.gg/ppy) di kanal `#osu-wiki`.
+
+## Memulai
+
+### Mendaftar
+
+1\. [Mendaftar](https://github.com/join) akun GitHub jika anda belum memilikinya.
+
+2\. [Masuk](https://github.com/login).
+
+3\. Lanjutkan ke [Forking](#forking).
+
+### Forking
+
+1\. Kunjungi [repo `osu-wiki`](https://github.com/ppy/osu-wiki).
+
+2\. Klik `Fork`.
+
+![](img/fork.jpg)
+
+3\. Klik `Fork` sekali lagi untuk diarahkan ke fork anda.
+
+4\. Jika anda melihat sesuatu yang mirip dengan gambar di bawah ini berarti anda telah membuat fork dari repo `ppy/osu-wiki` dan sekarang anda berada di fork anda.
+
+![](img/forked.jpg)
+
+5\. Lanjutkan ke [Syncing your fork](#syncing-your-fork).
+
+### Menyinkronkan fork anda
+
+1\. Kunjungi fork repo `osu-wiki` anda.
+
+2\. Di atas file, ada beberapa teks yang memberi tahu anda apakah branch dalam keadaan "behind", "ahead", atau "even dengan `ppy:master.` Jika branch dalam keadaan "behind" atau "ahead and behind" dengan jumlah behind-commits di belakangnya, branch anda belum diperbaharui. 
+
+![](img/fork-even.jpg "OK \(Tidak ada commit dan sudah menggunakan versi terbaru\)")
+
+![](img/fork-ahead.jpg "OK \(Ini adalah commit yang anda lakukan\)")
+
+![](img/fork-behind.jpg "Berpotensi buruk \(Branch anda belum diperbaharui\)")
+
+![](img/fork-ahead-behind.jpg "Berpotensi buruk \(Branch anda belum diperbaharui dengan commit yang anda lakukan\)")
+
+3\. Tidak jadi masalah besar, kecuali, jika anda akan menyunting file yang sudah disunting yang mungkin anda akan tertinggal dengan semakin banyaknya commit yang dibuat.
+
+4\. Untuk mengatasi ini, lihat [bagian Branch saya belum diperbaharui! dalam artikel Masalah Umum.](/wiki/owcg/Common_Issues/#my-branch-is-out-of-date!).
+
+5\. Ketika anda selesai dengan langkah 4, lanjutkan ke [Menyunting dari daring ataupun luring](#editing-online-or-locally).
+
+## Menyunting dari daring ataupun luring
+
+Pada titik ini, anda memiliki dua pilihan:
+
+- [GitHub Web Interface](/wiki/owcg/GitHub_Web_Interface) - menyunting dari daring; pilihan terbaik untuk penyuntingan artikel tunggal
+- [GitHub Desktop](/wiki/owcg/GitHub_Desktop) - menyunting dari luring; pilihan terbaik untuk penyuntingan artikel tunggal dan/atau ganda (juga termasuk mengunggah, menghapus, dan memindahkan gambar atau file)
+
+*Catatan: Anda tidak dibatasi dengan kedua pilihan yang tercantum di atas untuk penyuntingan.* Untuk menjaga panduan ini singkat dan sederhana, artikel-artikel ini tidak akan membahas penggunaan alat-alat yang digunakan secara mendalam. Terdapat aplikasi pihak ketiga lainnya yang dapat melakukan lebih banyak atau lebih sedikit dari apa yang sudah dilakukan oleh GitHub Web Interface dan GitHub Desktop.
+
+**Ketika anda telah menentukan pilihan antara *GitHub Web Interface* atau *GitHub Desktop*, anda dapat melanjutkan ke bagian selanjutnya.**
```

Comment:
it might be worth adding the translation for "web interface" and "desktop" in brackets after the english names